content/concept-history-sharding.md

```diff
@@ -0,0 +1,57 @@
+# History Sharding
+
+Rippled sharding is a feature that manages storing and sharing of distributed historical data, as shards. As servers operate, they naturally produce a database containing the nodes from the ledgers they witnessed or acquired during network runtime.
+
+Ledger data is stored by the node store, but the online delete logic rotates these databases when their size exceeds configured space limitations. In this way, Rippled sharding provides a middle ground between running a rippled server with full-history and one with only recent ledger data. History shards do not replace the node store, but implement a reliable path towards distributed ledger history across the rippled network.
+
+[![XRP Ledger Network: Nodes and Shard Store Diagram](img/xrp-ledger-network-node-and-shard-stores.png)](img/xrp-ledger-network-node-and-shard-stores.png)
+
+## Acquiring and Sharing Shard Stores
+
+Acquiring shards begins after synchronizing with the network and performing ledger backfilling. During this time of lower network activity, the shard database may be asked to select a shard by the acquire ledger process. If a shard is selected, the ledger acquire process begins by retrieving the sequence of the last ledger in the shard and working backwards toward the first. Shards continue to be acquired until reaching the maximum allocated disk space for shards, at which point the current shard may replace an older shard.
```

Comment:
> at which point the current shard may replace an older shard.

Is that true @miguelportilla ? Does that mean if you are trying to acquire shard, but run out of space before you finish it, you throw it away and try another one?

Reply:
@bachase A shard being acquired is not thrown away if the system runs out of space. Instead, progress will be halted until space becomes available. Logic dealing with the current shard replacing an older shard has not been implemented yet.  `current shard` meaning the latest complete shard, consisting of the newest 16384 ledgers.